modules/log/module.go

```diff
@@ -78,3 +78,26 @@ func (m *Module) Validate(r *domain.Resource) error {
 	}
 	return nil
 }
+
+func (m *Module) Act(r *domain.Resource, action string, params map[string]interface{}) (map[string]interface{}, error) {
+	switch action {
+	case "escalate":
+		r.Configs["log_level"] = increaseLogLevel(r.Configs["log_level"].(string))
+	}
+	return r.Configs, nil
+}
+
+func increaseLogLevel(currentLevel string) string {
```

Comment:
this function looks not really `increasing` log but just `setting log`.
what if we just name it as `setLogLevel`?

Reply:
Oh i see, in that case `increaseLogLevel` sounds simpler 😄